sr_tools/scripts/calib_strain_gauges.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+
+import os
+import rospy
+import numpy
+import csv
+from diagnostic_msgs.msg import DiagnosticArray
+
+
+class Calib_gauges():
+    def __init__(self, initial_weight, incremental_weight, final_weight):
+
+        self.initial_weight = initial_weight
+        self.incremental_weight = incremental_weight
+        self.final_weight = final_weight
+        self.motor_id = 0
+        self.number_of_tests = (self.final_weight-self.initial_weight)/self.incremental_weight
+        self.strain_gauges_id = 0
+        self.measurement = []
+        self.clean_measurement = []
+        self.headline_0 = []
+        self.temp_0 = []
+        self.headline_1 = []
+        self.temp_1 = []
+
+    def data_callback(self, data):
+
+        for i,value in enumerate(data.status): # iterate over the DiagnosticStatus message
+
+            fields = {}
+
+            for key,s in enumerate(value.values): # iterate over values array and store them in a dictionary with index and relative value
+
+                fields[i,s.key] = s.value
+           
+            for index, motor_id in fields.items(): # iterate over dictionary elements to search the wanted motor_id and the relative strain_gauge value
+
+                if fields.get((index[0],'Motor ID')) == "%s" % self.motor_id:
+                    if self.strain_gauges_id == 0:
+                        self.measurement.append(fields.get((index[0], 'Strain Gauge Left'))) #store read value in measurement array
```

Comment:
is this what you agreed with Luke?
I though we said Left and Right are confusing and we should call them directly 0 and 1.